scheduler/src/cook/mesos/sandbox.clj

```diff
@@ -36,6 +36,14 @@
 (def sandbox-updater-publish-rate (meters/meter ["cook-mesos" "scheduler" "sandbox-updater-publish-rate"]))
 (def sandbox-updater-tx-duration (timers/timer ["cook-mesos" "scheduler" "sandbox-updater-tx-duration"]))
 (def sandbox-updater-tx-rate (meters/meter ["cook-mesos" "scheduler" "sandbox-updater-tx-rate"]))
+(def sandbox-updater-unfiltered-count (counters/counter ["cook-mesos" "scheduler" "sandbox-updater-unfiltered-count"]))
+
+(defn- threadsafe-counter-reset
```

Comment:
Why is the word `threadsafe` in the function name?

Reply:
Removed, I intended to name it thread unsafe :( 
I updated the docstring to reflect this unsafe behavior.